github/pulls.go

```diff
@@ -174,6 +181,7 @@ type NewPullRequest struct {
 	Body                *string `json:"body,omitempty"`
 	Issue               *int    `json:"issue,omitempty"`
 	MaintainerCanModify *bool   `json:"maintainer_can_modify,omitempty"`
+	NodeID              *string `json:"node_id,omitempty"`
```

Comment:
I don't think `NewPullRequest` struct should get the `NodeID` field, should it?

It's a struct we use to pass parameters to [`PullRequests.Create`](https://godoc.org/github.com/google/go-github/github#PullRequestsService.Create) method. You can't specify the `node_id` when creating a new PR (i.e., https://developer.github.com/v3/pulls/#create-a-pull-request).

Reply:
Ah, sorry about that, I'd misread the flow in Create - `NewPullRequest` goes in, regular `PullRequest` comes out. Fixed.